src/ripple/app/paths/PathRequest.h

```diff
@@ -91,13 +89,18 @@ class PathRequest : public std::enable_shared_from_this<PathRequest>,
     doCreate(std::shared_ptr<RippleLineCache> const&, Json::Value const&);
 
     Json::Value
-    doClose(Json::Value const&);
+    doClose() override;
     Json::Value
-    doStatus(Json::Value const&);
+    doStatus(Json::Value const&) override;
+    void
+    doAborting() const;
 
     // update jvStatus
     Json::Value
-    doUpdate(std::shared_ptr<RippleLineCache> const&, bool fast);
+    doUpdate(
+        std::shared_ptr<RippleLineCache> const&,
+        bool fast,
+        std::function<bool(void)> continueCallback = {});
```

Comment:
Consider passing `function` by `const&`. Here's a patch to do that if you want: https://github.com/seelabs/rippled/commit/97ede1ef2530d6dd4fba87bb6a5ef43cc02cf5ac

Reply:
Will do.